python/ccxt/async_support/base/ws/cache.py

```diff
@@ -47,6 +47,10 @@ def __getitem__(self, item):
         else:
             return deque[item]
 
+    def __getattr__(self, attr_name):
+        # support snake_case
+        if (attr_name == 'get_limit'):
+            return self.getLimit
 
```

Comment:
ast transpiler causes the `getLimit` method to be transpiled int snake-case, so this was easiest way to fix

Reply:
we can do this or use a regex

Reply:
same question here, why not `def get_limit` explicit is better